src/microbe_stage/gui/ExtinctionBox.tscn

```diff
@@ -26,6 +26,7 @@ tracks/0/keys = {
 }
 
 [node name="ExtinctionBox" type="Popup"]
+modulate = Color( 1, 1, 1, 0 )
```

Comment:
Is this an unintended change? Seems a bit suspicious to be modifying the extinction screen in this PR.